docs/dev/libs/state_processor_api.md

```diff
@@ -238,9 +238,11 @@ This means if an operator contains a stateful process function such as:
 public class StatefulFunctionWithTime extends KeyedProcessFunction<Integer, Integer, Void> {
  
    ValueState<Integer> state;
- 
+   ValueStateDescriptor<Integer> stateDescriptor;
+   
    @Override
    public void open(Configuration parameters) {
+	  stateDescriptor = new ValueStateDescriptor<>("state", Types.INT);
```

Comment:
Nit: can we align these two lines.
In other word, do we need to align all the StateDescriptor creating?